backend/app/services/filter_service.rb

```diff
@@ -0,0 +1,56 @@
+class FilterService < ApplicationService
```

Comment:
can we use this service here : https://github.com/betagouv/datapass/blob/chore/enrollments-public-listing-pagination/backend/app/controllers/users_controller.rb#L13-L38 ?